livekit-ffi/protocol/room.proto

```diff
@@ -89,7 +89,7 @@ message PublishDataRequest {
   uint64 data_ptr = 2;
   uint64 data_len = 3;
   DataPacketKind kind = 4;
-  repeated string destination_sids = 5; // destination
```

Comment:
also not sure what the right approach here would be, but we could of course "only" deprecate destination_sids for now and introduce `destination_identites=7`

Reply:
would that not be as breaking of a change, as implementations would now require `destination_identities=7` as well?

Reply:
all proto fields are pre-filled with an empty value, I think. So the default would be used in case it was missing entirely. 
In my understanding it's best not to re-use the field index, see https://protobuf.dev/programming-guides/proto3/#consequences

Reply:
got it

Reply:
you could add `repeated string destination_sids = 5 [deprecated=true];` to indicate the deprecation